src/utilities/enums.ts

```diff
@@ -0,0 +1,5 @@
+export enum ListStatus {
+  CLOSED,
+  FOLLOWERS,
+  FOLLOWING,
+}
```

Comment:
Even though it's used in a few different places I would keep this in the FollowersFollowing component. Since it's specific to that component and only used by directly adjacent components. I feel we should use the utilities for types that are used all over the place.

Reply:
@wilwade what do you think about this?

Reply:
@wilwade what do you think?

Reply:
Still think that splitting up this into it's own entire file just to see it be used in two related components seems unnecessary. It'd be cleaner if the Parent component of the two made it and the Child imported it.